### ISSUE TITLE
npx lefthook throwing `command not found`

### ISSUE THREAD

**Issue by luizwhite** [evilmartians/lefthook#773]
### :wrench: Summary

It seems after lefthook 1.6.18, when git hooks are run an error occurs with the message `sh: lefthook: command not found`.

### Lefthook version

```bash
❯ npx lefthook version -f
sh: lefthook: command not found
```
### Steps to reproduce

```bash
❯ npx lefthook version -f
sh: lefthook: command not found
```
### Expected results

```bash
❯ npx lefthook@1.6.18 version -f
1.6.18 4ea200b04d374ba1bb9f66a940430309a2838d27
```
### Actual results

```bash
❯ npx lefthook version -f
sh: lefthook: command not found
```
### Possible Solution

No solution, since when lefthook creates git hooks, it falls in this `elif`:
```bash
  elif command -v npx >/dev/null 2>&1
  then
    npx lefthook "$@"
```

So, it is not possible in this case to set lefthook version, correct?

### Logs / Screenshots

```bash
❯ LEFTHOOK_VERBOSE=true npx lefthook version -f
sh: lefthook: command not found
```

### node/npm

```bash
❯ node --version
v20.15.1
❯ npm --version
10.8.2
```

**Comment by luizwhite** [comment]
Hi @mrexox !
The error persists

**Comment by mrexox** [comment]
Yes, `npx lefthook` is still not working for some reason. I am investigating.

**Comment by mrexox** [comment]
I've fixed the issue with 1.7.4 release!
